pkg/cmd/admin/migrate/images/imagerefs_test.go

```diff
@@ -0,0 +1,601 @@
+							Containers: []kapi.Container{
+								{Image: "index.docker.io/foo/bar"},
+								{Image: "index.docker.io/foo/bar"},
+							},
+						},
+					},
+				},
+				{
+					obj: &kapi.ReplicationController{
+						Spec: kapi.ReplicationControllerSpec{
+							Template: &kapi.PodTemplateSpec{
+								Spec: kapi.PodSpec{
+									Containers: []kapi.Container{
+										{Image: "docker.io/foo/bar"},
+										{Image: "foo/bar"},
+									},
+								},
+							},
+						},
+					},
+					changed: true,
+					expected: &kapi.ReplicationController{
+						Spec: kapi.ReplicationControllerSpec{
+							Template: &kapi.PodTemplateSpec{
+								Spec: kapi.PodSpec{
+									Containers: []kapi.Container{
+										{Image: "index.docker.io/foo/bar"},
+										{Image: "index.docker.io/foo/bar"},
+									},
+								},
+							},
+						},
+					},
+				},
+				{
+					obj: &kextensions.Deployment{
+						Spec: kextensions.DeploymentSpec{
+							Template: kapi.PodTemplateSpec{
+								Spec: kapi.PodSpec{
+									Containers: []kapi.Container{
+										{Image: "docker.io/foo/bar"},
+										{Image: "foo/bar"},
+									},
+								},
+							},
+						},
+					},
+					changed: true,
+					expected: &kextensions.Deployment{
+						Spec: kextensions.DeploymentSpec{
+							Template: kapi.PodTemplateSpec{
+								Spec: kapi.PodSpec{
+									Containers: []kapi.Container{
+										{Image: "index.docker.io/foo/bar"},
+										{Image: "index.docker.io/foo/bar"},
+									},
+								},
+							},
+						},
+					},
+				},
+				{
+					obj: &deployapi.DeploymentConfig{
+						Spec: deployapi.DeploymentConfigSpec{
+							Template: &kapi.PodTemplateSpec{
+								Spec: kapi.PodSpec{
+									Containers: []kapi.Container{
+										{Image: "docker.io/foo/bar"},
+										{Image: "foo/bar"},
+									},
+								},
+							},
+						},
+					},
+					changed: true,
+					expected: &deployapi.DeploymentConfig{
+						Spec: deployapi.DeploymentConfigSpec{
+							Template: &kapi.PodTemplateSpec{
+								Spec: kapi.PodSpec{
+									Containers: []kapi.Container{
+										{Image: "index.docker.io/foo/bar"},
+										{Image: "index.docker.io/foo/bar"},
+									},
+								},
+							},
+						},
+					},
+				},
+				{
+					obj: &kextensions.DaemonSet{
+						Spec: kextensions.DaemonSetSpec{
+							Template: kapi.PodTemplateSpec{
+								Spec: kapi.PodSpec{
+									Containers: []kapi.Container{
+										{Image: "docker.io/foo/bar"},
+										{Image: "foo/bar"},
+									},
+								},
+							},
+						},
+					},
+					changed: true,
+					expected: &kextensions.DaemonSet{
+						Spec: kextensions.DaemonSetSpec{
+							Template: kapi.PodTemplateSpec{
+								Spec: kapi.PodSpec{
+									Containers: []kapi.Container{
+										{Image: "index.docker.io/foo/bar"},
+										{Image: "index.docker.io/foo/bar"},
+									},
+								},
+							},
+						},
+					},
+				},
+				{
+					obj: &kextensions.ReplicaSet{
```

Comment:
I'm not going to mention it until it's supported.  If we don't land it, it
would be wrong.

On Wed, Jul 27, 2016 at 4:06 PM, Maciej Szulik <notifications@github.com>
wrote:

> In pkg/cmd/admin/migrate/images/imagerefs_test.go
> <https://github.com/openshift/origin/pull/9240#discussion_r72512914>:
>
> > +					changed: true,
> > +					expected: &kextensions.DaemonSet{
> > +						Spec: kextensions.DaemonSetSpec{
> > +							Template: kapi.PodTemplateSpec{
> > +								Spec: kapi.PodSpec{
> > +									Containers: []kapi.Container{
> > +										{Image: "index.docker.io/foo/bar"},
> > +										{Image: "index.docker.io/foo/bar"},
> > +									},
> > +								},
> > +							},
> > +						},
> > +					},
> > +				},
> > +				{
> > +					obj: &kextensions.ReplicaSet{
>
> You didn't mention ReplicaSets in the description of the command.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9240/files/1ddd7d050a21f14d07b8d4198df6e7eec9cf8475#r72512914>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_pwVYx9nwjPVewCyt20P9XuOtrWidks5qZ7pWgaJpZM4IxnZA>
> .
>